packages/node/src/indexer/entities/Poi.entity.ts

```diff
@@ -37,7 +39,7 @@ export function PoiFactory(sequelize: Sequelize, schema: string): PoiRepo {
       },
       parentHash: {
         type: DataTypes.BLOB,
-        allowNull: true,
+        allowNull: false,
```

Comment:
first block doesn't have parentHash, are you planning to use `0x00` for the first block?